### PR TITLE
Update documentation for issue #463 #460 #474

### PR DESCRIPTION
**Pull request by straithe**

### CODE REVIEW

File: docs/source/programming.rst [hunk 1]
```diff
@@ -6,6 +6,8 @@ This page describes how to load compiled firmware onto an Ubertooth One. Other p
 
 There are three ways to program an Ubertooth board: the USB bootloader, the ISP bootloader, and JTAG.
 
+Please note that it is not possible to program the board's usb firmware via ubertooth-dfu as there is software protection in place.
```
Comment (mossmann) [inline docs/source/programming.rst:9]: Please change "program" to "replace" and change "usb firmware" to "USB bootloader".

File: docs/source/build_guide.rst [hunk 2]
```diff
@@ -69,7 +69,7 @@ Mac OS X users can use either MacPorts or Homebrew to install the required packa
 
 		brew install libusb wget cmake pkg-config
 		or
-		sudo port install libusb wget cmake python38 py38-numpy py38-qtpy
+		sudo port install libusb wget cmake python27 py27-numpy py27-pyside
```
Comment (mossmann) [inline docs/source/build_guide.rst:72]: This change should be removed. I'm closing #463 as it was incorrect.

**Review by mossmann — CHANGES_REQUESTED** [review]
a few small changes requested

File: docs/source/programming.rst (hunk 1 above)
Comment (straithe) [reply docs/source/programming.rst]: Done.

File: docs/source/build_guide.rst (hunk 2 above)
Comment (straithe) [reply docs/source/build_guide.rst]: Done

**Pull request merged**